delfin/task_manager/manager.py

```diff
@@ -39,6 +49,49 @@ def __init__(self, service_name=None, *args, **kwargs):
         self.alert_sync = alerts.AlertSyncTask()
         super(TaskManager, self).__init__(*args, **kwargs)
 
+    def periodic_performance_collect(self):
+        LOG.info("Scheduled perf-sync operation starting.")
+
+        # Load the scheduler configuration file
+        try:
+            with open(CONF.scheduler.config_path) as f:
+                data = json.load(f)
+        except FileNotFoundError as e:
+            raise exception.ConfigNotFound(e)
+
+        # create the object of periodic scheduler
+        schedule = BlockingScheduler()
+
+        # create the objet to StorageController class, so that
+        # methods of that class can be called by scheduler
+        storage_cls = StorageController()
```

Comment:
How about having performnace releated methods in new class 'StoragePerfromanceController' ?